libs/template/labs/src/i18n/translations/en.ts

```diff
@@ -23,6 +23,16 @@ const cart = {
   'oauth.logging-you-in': 'Logging you in...',
 };
 
+const coupon = {
+  'coupon.have-a-coupon': 'Have a coupon?',
+  'coupon.insert': 'Please insert a coupon code',
+  'coupon.invalid': 'Coupon code can not be added',
+  'coupon.apply': 'Apply',
```

Comment:
not needed, as it will resolve to "Apply" automatically. Same for coupon-code and successfully-applied